kapitan/cli.py

```diff
@@ -257,8 +256,7 @@ def main():
                 yaml.dump(inv, sys.stdout, Dumper=PrettyDumper, default_flow_style=False)
         except Exception as e:
             if not isinstance(e, KapitanError):
-                logger.error("\n~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~")
-                traceback.print_exc()
+                logger.exception("\n~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~")
```

Comment:
does logger.exception include the same as traceback?

Reply:
Based on this https://docs.python.org/3.7/library/logging.html#logging.Logger.exception it also includes the stacktrace and I think is more pythonic (?).